wolfssl/wolfcrypt/memory.h

```diff
@@ -214,7 +214,14 @@ WOLFSSL_API int wolfSSL_GetAllocators(wolfSSL_Malloc_cb* mf,
     typedef struct wc_Memory wc_Memory; /* internal structure for mem bucket */
     typedef struct WOLFSSL_HEAP {
         wc_Memory* ava[WOLFMEM_MAX_BUCKETS];
+    #ifndef WOLFSSL_LEAN_STATIC_MEMORY
```

Comment:
Please document this new build option `WOLFSSL_LEAN_STATIC_MEMORY` at the top of memory.c or in the header. Does it reduce code size, change pools? Does it also require use with WOLFSSL_STATIC_MEMORY... etc.